content/applications/inventory_and_mrp/purchase/manage_deals/control_bills.rst

```diff
@@ -1,110 +1,129 @@
-
-If you choose *On received quantities* instead, you can only create one once you have at least
-received some of the products you have ordered. The quantities you have received are used to
-generate the draft bill. If you try to create one without having received any product, you get an
-error message.
-
-.. image:: control_bills/control-bills-no-invoiceable-line.png
-   :align: center
-   :alt: No invoiceable line error message in Odoo Purchase
-
-Change a specific product's control policy
-------------------------------------------
-
-If you want to modify a specific product's control policy, go to :menuselection:`Products -->
-Products`, open it, click on *Edit* and go to the *Purchase tab*. There you can change a product's
-default bill control policy.
-
-.. image:: control_bills/control-bills-quantities-on-product.png
-   :align: center
-   :alt: Vendor bills default control setting for new products in Odoo Purchase
-
-View a purchase order's billing status
-======================================
-
-Once you confirm an order, you can view its *Billing Status* by going to the *Other Information*
-tab.
+- :guilabel:`Ordered quantities`: creates a vendor bill as soon as a purchase order is confirmed.
+  The products and quantities in the purchase order are used to generate a draft bill.
+- :guilabel:`Received quantities`: a bill is created only *after* part of the total order has been
+  received. The products and quantities *received* are used to generate a draft bill. An error
+  message will appear if creation of a vendor bill is attempted without receiving anything.
 
-.. image:: control_bills/control-bills-billing-status.png
+.. image:: control_bills/bill-control-policy-error-message.png
    :align: center
-   :alt: Purchase order billing status in Odoo Purchase
+   :alt: Bill control policy draft bill error message.
 
-Below you can find a list of the different *Billing Status*, and when they are displayed, depending
-on the products' bill control policy.
+.. note::
+   If one or two products need a different control policy, the default bill control setting can be
+   overridden by going to the :guilabel:`Purchase` tab in a product's template and modifying its
+   :guilabel:`Control Policy` field.
+
+Example flow: Ordered quantities
+--------------------------------
+
+To complete an example workflow using the *ordered quantities* bill control policy, first go to
+:menuselection:`Purchase --> Configuration --> Settings`, scroll down to the :guilabel:`Invoicing`
+section, and select :guilabel:`Ordered quantities`. Then, :guilabel:`Save` changes.
+
+In the :guilabel:`Purchase` app, create a new :abbr:`RFQ (Request for Quotation)`. Fill out the
+information on the quotation, add products to the invoice lines, and click
+:guilabel:`Confirm Order`. Then, click :guilabel:`Create Bill`. Since the policy is set to *ordered
+quantities*, the draft bill can be confirmed as soon as it is created, without any products
+actually being received.
```

Comment:
100th char line break (can break in the middle of guilabel tag if there's a space) and adding the word `form` here.
```suggestion
information on the quotation form, add products to the invoice lines, and click :guilabel:`Confirm
Order`. Then, click :guilabel:`Create Bill`. Since the policy is set to *ordered quantities*, the
draft bill can be confirmed as soon as it is created, without any products actually being received.
```